source/wp-content/themes/wporg-developer-2023/stylesheets/page-dashicons.css

```diff
@@ -11,14 +11,20 @@
   box-sizing: border-box;
```

Comment:
Can you run `yarn lint:css source/wp-content/themes/wporg-developer-2023/stylesheets/page-dashicons.css --fix` to fix the indentation issues in this file, since it was generated & uses the wrong style?